client/src/components/Allocations/AllocationsLayout.js

```diff
@@ -8,30 +8,40 @@ import {
 } from 'react-router-dom';
 import { Nav, NavItem, NavLink } from 'reactstrap';
 import { string } from 'prop-types';
-import { Icon, LoadingSpinner } from '_common';
+import { Icon, LoadingSpinner, Section, SectionTable } from '_common';
 import { AllocationsTable } from './AllocationsTables';
 import { AllocationsRequestModal } from './AllocationsModals';
 import * as ROUTES from '../../constants/routes';
 
 export const Header = ({ page }) => {
-  const root = `${ROUTES.WORKBENCH}${ROUTES.ALLOCATIONS}/${page}`;
   return (
-    <div id="allocations-header">
-      <div id="header-text">
+    /* !!!: Temporary extra markup to make simpler PR diff */
+    /* eslint-disable prettier/prettier */
+    <>
         <Link to={`${ROUTES.WORKBENCH}${ROUTES.ALLOCATIONS}`}>Allocations</Link>
         <span>&nbsp;/&nbsp;</span>
         <span>{page[0].toUpperCase() + page.substring(1)}</span>
-      </div>
+    </>
+    /* eslint-enable prettier/prettier */
+  );
+};
+Header.propTypes = { page: string.isRequired };
+
+export const Actions = ({ page }) => {
```

Comment:
__Explanation__
I broke up the `<Header>` into:
- `<Header>` (for `<Section header>`)
- `<Actions>` (for `<Section headerActions>`)